core/templates/dev/head/components/profile-link-directives/profile-link-text.directive.ts

```diff
@@ -33,8 +33,10 @@ angular.module('oppia').directive('profileLinkText', [
       controllerAs: '$ctrl',
       controller: [function() {
         var ctrl = this;
-        this.isUsernameLinkable = function(username) {
-          return SYSTEM_USER_IDS.indexOf(username) === -1;
+        this.$onInit = function() {
+          this.isUsernameLinkable = function(username) {
```

Comment:
Shouldn't this be "ctrl" instead of "this"?

Reply:
var ctrl = this; 
this.$onInit or ctrl.$onInit must be the same. i used this for better understanding.
I'll make it ctrl.$onInit?

Reply:
No, I meant, ctrl.isUsernameLinkable. In JavaScript, the meaning of "this" can change depending on which scope it's in.

Reply:
correct, i'll update this Thanks!